hpe3parclient/client.py

```diff
@@ -4816,3 +4731,242 @@ def getScheduleStatus(self, schedule_name):
+            conversion is in progress.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - INV_OPERATION_VV_NOT_IN_NORMAL_STATE - Volume state is not normal
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - INV_OPERATION_VV_PEER_VOLUME - Cannot modify a peer volume.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPConflict`
+            - INV_OPERATION_VV_TASK_CANCEL_IN_PROGRESS - Invalid operation:
+            A task involving the volume is being canceled..
+        :raises: :class:`~hpe3parclient.exceptions.HTTPConflict`
+            - INV_OPERATION_VV_PROMOTE_IN_PROGRESS - Invalid operation: Volume
+            promotion is in progress.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPConflict`
+            - INV_OPERATION_VV_TUNE_IN_PROGRESS - Invalid operation: Volume
+            tuning is in progress.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPBadRequest`
+            - NO_SPACE - Not Enough space is available
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - NODE_DOWN - The node is down.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPNotFound`
+            - NON_EXISTENT_CPG - The CPG does not exists.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPNotFound`
+            - NON_EXISTENT_VOL - volume doesn't exist
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - VV_IN_INCONSISTENT_STATE - The volume has an internal consistency
+            error.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - VV_IS_BEING_REMOVED - The volume is being removed.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - VV_NEEDS_TO_BE_CHECKED - The volume needs to be checked.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - VV_NOT_STARTED - Volume is not started.
+        :raises: :class:`~hpe3parclient.exceptions.HTTPForbidden`
+            - INV_INPUT_VV_IS_FPVV - A fully provisioned volume cannot be
+            compressed.
+
+        """
+        info = {'action': self.TUNE_VOLUME, 'tuneOperation': tune_operation}
+        if optional is not None and not self.compression_supported:
+            if 'compression' in optional.keys():
+                del optional['compression']
+        if optional:
+            if self.primera_supported:
+                if optional.get('conversionOperation') == 3 \
+                        and optional.get('compression') is True:
+                    optional['conversionOperation'] = 4
+
+                if (optional.get('conversionOperation') == 3 and
+                        optional.get('compression') is False) or\
+                   (optional.get('conversionOperation') == 1 and
+                        optional.get('compression') is True):
+                    raise exceptions.HTTPBadRequest("invalid input: For Deco\
```

Comment:
Please replace all occurrence of the word "Deco volume" with "Compression and deduplicated volume"